components/bigcommerce/bigcommerce.app.mjs

```diff
@@ -1,11 +1,90 @@
+import { axios } from "@pipedream/platform";
+import constants from "./common/constants.mjs";
 export default {
   type: "app",
   app: "bigcommerce",
-  propDefinitions: {},
+  propDefinitions: {
+    storeHash: {
+      type: "string",
+      label: "Store Hash",
+      description: "Your store hash.",
+    },
+    channel: {
+      type: "boolean",
+      label: "Using channel",
+      description: "Channel of the webhook",
+    },
+    channelId: {
+      type: "string",
+      label: "Channel Id",
+      description: "Id of the channel",
+    },
```

Comment:
I see this is used in `additionalProps()`, so I think it doesn't need to be defined here